sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md

```diff
@@ -243,7 +243,7 @@ And to receive a batch:
 ServiceBusReceiver receiver = client.CreateReceiver(queueName);
 
 // the received message is a different type as it contains some service set properties
-IList<ServiceBusReceivedMessage> receivedMessages = await receiver.ReceiveBatchAsync(maxMessages: 2);
+IList<ServiceBusReceivedMessage> receivedMessages = await receiver.ReceiveMessagesAsync(maxMessages: 2);
```

Comment:
We need to update the same [here](https://github.com/Azure/azure-sdk-for-net/blob/332e4375d0805bbe016a3cce5cbb8ebe93f03e23/sdk/servicebus/Azure.Messaging.ServiceBus/MigrationGuide.md#receiving-messages).